resources/model/OpenStudio.idd

```diff
@@ -14885,11 +15040,63 @@ OS:Chiller:Electric:EIR,
+  A17, \field End-Use Subcategory
        \note Any text may be used here to categorize the end-uses in the ABUPS End Uses by Subcategory table.
        \type alpha
        \retaincase
        \default General
+  A18, \field Condenser Flow Control
+       \note Select the chiller condenser flow request mode. With "ConstantFlow" a chiller will always request
+       \note its maximum condenser flow rate. With "ModulatedChillerPLR" the condenser flow request corresponds
+       \note to the chiller part load ratio multiplied by the chiller maximum condenser flow rate.  With 
+       \note "ModulatedLoopPLR" the chiller will request a flow rate that is function of the chilled water 
+       \note loop's part load ratio, see the "Condenser Loop Flow Rate Fraction Function of Loop Part Load Ratio
+       \note Curve Name" input. With "ModulatedDeltaTemperature" the chiller will request the flow rate required to meet 
+       \note the condenser loop load based on the condenser leaving fluid temperature and a reference temperature,
+       \note see the "Temperature Difference Across Condenser"  and "Temperature Difference Across Condenser Schedule
+       \note Name" input.
+       \note Use "ConstantFlow" when modeling a constant flow condenser plant loop, choose one of the other inputs
+       \note when modeling a variable flow condenser plant loop.
+       \type choice
+       \key ConstantFlow
+       \key ModulatedChillerPLR
+       \key ModulatedLoopPLR
+       \key ModulatedDeltaTemperature
+       \required-field
+  A19, \field Condenser Loop Flow Rate Fraction Function of Loop Part Load Ratio Curve Name
+       \note Condenser loop flow rate fraction as a function of loop part load ratio
+       \note CWFR = C * PLR + D 
+       \note Where:
+       \note CWFR is the condenser water flow fraction (actual/design)
+       \note C and D are coefficients, see "Optimizing Design & Control Of Chilled Water Plants, Part 5", S. Taylor, ASHRAE Journal June 2012
+       \note PLR is the chilled water plant loop part load ratio (actual/design)
+       \type object-list
+       \object-list UnivariateFunctions
+  A20, \field Temperature Difference Across Condenser Schedule Name
+       \note A schedule that defines the temperature difference across the condenser. This input is used to 
+       \note calculate the condenser flow. This input is only used when "Condenser Flow Control" is set to
+       \note "ModulatedDeltaTemperature".
+       \type object-list
+       \object-list ScheduleNames
+  N19, \field Condenser Minimum Flow Fraction
+       \note This input corresponds to the minimum flow fraction to be simulated. The minimum condenser flow
+       \note corresponds to this fraction multiplied by the maximum condenser flow rate. This input is only used
+       \note when the "Condenser Flow Control" input is set to "ModulatedChillerPLR", "ModulatedLoopPLR" or
+       \note "ModulatedDeltaTemperature".
+       \type real
```

Comment:
I suggest making this a required field, hardcode to 0.2 in Ctor

cf https://github.com/NREL/EnergyPlus/pull/10511#pullrequestreview-2342091814